sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -179,6 +173,11 @@ unsupportedHiveNativeCommands
     | kw1=ALTER kw2=TABLE tableIdentifier kw3=TOUCH
     | kw1=ALTER kw2=TABLE tableIdentifier partitionSpec? kw3=COMPACT
     | kw1=ALTER kw2=TABLE tableIdentifier partitionSpec? kw3=CONCATENATE
+    | kw1=START kw2=TRANSACTION
+    | kw1=COMMIT
+    | kw1=ROLLBACK
+    | kw1=DFS
```

Comment:
@yhuai If we do not disable them, the CLI driver will pass these commands to Hive. Below is the related code: https://github.com/gatorsmile/spark/blob/84aa14a5deda14083520e8e23f83cdb7f5bbb2bc/sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala#L380-L384

 It will be different from what we want, I think. To make the behaviors consistent, I submitted the following PR: https://github.com/apache/spark/pull/13122

Let me know if you have any concern. Thanks!